### ISSUE TITLE
Add more effects?

### ISSUE THREAD

**Issue by antiboredom** [antiboredom/zoom-escaper#1]
A bit on the fence, but maybe I should add some more effects?

**Comment by Rillke** [comment]
Maybe [different toilets flushing](https://commons.wikimedia.org/wiki/Category:Sounds_of_toilets)? Or [somebody eating Pizza](https://commons.wikimedia.org/wiki/Category:Sounds_of_cooking)? Note, however, that most sounds from [WMCommons](https://commons.wikimedia.org/wiki/Category:Sounds_by_type) require attribution and mention of the license.

**Comment by yogesh-7** [comment]
yes please add more effects like raining/thunder , door bell ring, girl crying,

**Comment by msbrown** [comment]
Some ideas for more sounds: the seemingly ever present leaf blowers, door knocking or door bell ringing, the sounds of a lot of pigeons as though outside a nearby window. Maybe some neighbor noises like muffled aerobics tape and thumping, vacuuming, blender.   Or add a custom sound feature: You connect to the sound and it give a simple volume control and option to loop the sound.

**Comment by fatihcandev** [comment]
I have a colleague and she has a pet parakeet. Literally every meeting, the animal makes high volume noises non stop. It feels like a jungle in there and it's a pretty annoying. My effect recommendation would be that 😄
